### PR TITLE
[NCL-6494] Fix NPM Brew push by bumping kojiji version

### PR DESCRIPTION
**Pull request by pkocandr**
### Checklist:

* [ ] Have you added a note in the CHANGELOG.md for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: core/src/main/java/org/jboss/pnc/causeway/brewclient/BrewClientImpl.java [hunk 1]
```diff
@@ -232,8 +232,12 @@ private boolean checkImportErrors(KojiImportResult result, ImportFileGenerator i
         if (kojiErrors != null) {
             for (Map.Entry<String, KojijiErrorInfo> e : kojiErrors.entrySet()) {
                 String artifactId = importFiles.getId(e.getKey());
-                log.warn("Failed to import artifact {} ({}): {}", artifactId, e.getKey(), e.getValue());
-                log.debug("Artifact import error:", e);
+                if (log.isWarnEnabled()) {
+                    KojijiErrorInfo errorInfo = e.getValue();
+                    String message = String
+                            .format("Failed to import artifact %s (%s): %s", artifactId, e.getKey(), errorInfo);
+                    log.warn(message, errorInfo.getError());
```
Comment (janinko) [inline core/src/main/java/org/jboss/pnc/causeway/brewclient/BrewClientImpl.java:239]: Is KojijiErrorInfo an exception?

**Comment by pkocandr** [comment]
retest this please

File: core/src/main/java/org/jboss/pnc/causeway/brewclient/BrewClientImpl.java (hunk 1 above)
Comment (pkocandr) [reply core/src/main/java/org/jboss/pnc/causeway/brewclient/BrewClientImpl.java:239]: No, it's a DTO. But KojijiErrorInfo.error is.

**Comment by pkocandr** [comment]
retest this please

**Comment by pkocandr** [comment]
retest this please

**Review by janinko — APPROVED** [review]

**Pull request merged**